### PR TITLE
Release 1.17.1

### PR DESCRIPTION
**Pull request by serhii73**

### CODE REVIEW

**Review by wRAR — APPROVED** [review]

**Review by VMRuiz — APPROVED** [review]

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 =============
 
+1.17.1 (2023-01-05)
+-------------------
+
+bug: Adding the use of ItemAdapter to prevent assumptions of item nature (#358)
+bug: Fix Slack dependency name issue (#367)
+chore: Change the Ubuntu version on workflow settings (#373)
+docs: Adding PeriodicExecutionTimeMonitor to the batteries docs (#368)
+misc: Fix compatibility issues with jsonschema>=4 (#364)
```
Comment (rennerocha) [inline CHANGES.rst:11]: **suggestion** Add the links to the PRs and/or issues related to each of these new features as we already have in all other latest versions changelogs.

**Review by Gallaecio — APPROVED** [review]

**Review by proway2 — APPROVED** [review]
LGTM

**Comment by proway2** [comment]
We've been waiting for the new feature for so long :-)

@rennerocha would you please check it again?

**Review by rennerocha — APPROVED** [review]
LGTM!
After the merge do not forget to create 1.17.1 tag so the release in PyPI will be made automatically.

**Pull request merged**